sci-libs/calculix-ccx/calculix-ccx-2.10.ebuild

```diff
@@ -44,6 +48,11 @@ src_configure() {
 	if use threads; then
 		append-cflags "-DUSE_MT"
 	fi
+	
+	if use openmp; then
+		export FFLAGS="-fopenmp $(FFLAGS)"
```

Comment:
I googled it and didn't have any result, I thought it was just for C flags. 
In fact it is explained here: https://devmanual.gentoo.org/eclass-reference/flag-o-matic.eclass/index.html Thxs